Tests/Data/ThermoHydroMechanics/Linear/Point_injection/square_1e2.prj

```diff
@@ -0,0 +1,306 @@
+            </constitutive_relation>
+            <intrinsic_permeability>k</intrinsic_permeability>
+            <specific_storage>S</specific_storage>
+            <biot_coefficient>alpha</biot_coefficient>
+            <porosity>phi</porosity>
+            <solid_density>rho_sr</solid_density>
+			<fluid_density>rho_fr</fluid_density>
+			<fluid_viscosity>mu</fluid_viscosity>
+            <fluid_volumetric_thermal_expansion_coefficient>beta_f</fluid_volumetric_thermal_expansion_coefficient>
+            <solid_linear_thermal_expansion_coefficient>alpha_s</solid_linear_thermal_expansion_coefficient>
+            <solid_specific_heat_capacity>C_s</solid_specific_heat_capacity>
+            <solid_thermal_conductivity>lambda_s</solid_thermal_conductivity>
+            <fluid_specific_heat_capacity>C_f</fluid_specific_heat_capacity>
+            <fluid_thermal_conductivity>lambda_f</fluid_thermal_conductivity>
+            <reference_temperature>T0</reference_temperature>
+            <process_variables>
+                <displacement>displacement</displacement>
+                <pressure>pressure</pressure>
+                <temperature>temperature</temperature>
+            </process_variables>
+            <secondary_variables>
+                <secondary_variable type="static" internal_name="sigma" output_name="sigma"/>
+                <secondary_variable type="static" internal_name="epsilon" output_name="epsilon"/>
+            </secondary_variables>
+            <specific_body_force>0 0</specific_body_force>
+        </process>
+    </processes>
+    <time_loop>
+        <processes>
+            <process ref="THERMO_HYDRO_MECHANICS">
+                <nonlinear_solver>basic_newton</nonlinear_solver>
+                <convergence_criterion>
+                    <type>PerComponentDeltaX</type>
+                    <norm_type>NORM2</norm_type>
+                    <abstols>1e-5 1e-5 1e-5 1e-5</abstols>
+                </convergence_criterion>
+                <time_discretization><type>BackwardEuler</type></time_discretization>
+                <time_stepping>
+                    <type>FixedTimeStepping</type>
+                    <t_initial>0</t_initial>
+                    <t_end>1000</t_end>
+                    <timesteps>
+                        <pair><repeat>10</repeat>
+                        <delta_t>100</delta_t></pair>
+                    </timesteps>
+                </time_stepping>
+            </process>
+        </processes>
+        <output>
+            <type>VTK</type>
+            <prefix>square_1e0</prefix>
+            <timesteps>
+                <pair><repeat>1</repeat>
+                <each_steps>10</each_steps></pair>
+            </timesteps>
+                <variables>
+                    <variable>displacement</variable>
+                    <variable>pressure</variable>
+                    <variable>temperature</variable>
+                    <variable>sigma_xx</variable>
```

Comment:
Should be sigma and epsilon, not component-wise. Same in all other project files.

Reply:
Modified to sigma and epsilon.